Description: skip doctest
 The doctest depends on an arbitrary file on a random user's home.
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-10-30

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,2 +11,5 @@
 
+[lib]
+doctest = false
+
 [package]
